portal-frontend/src/app/features/edit-submission/proposal/roso-proposal/roso-proposal.component.ts

```diff
@@ -186,7 +197,7 @@ export class RosoProposalComponent extends StepComponent implements OnInit, OnDe
   onChangeNOI(selectedValue: string) {
     if (selectedValue === 'true') {
```

Comment:
Not sure if it will look better, but you could use parseStringToBoolean and work with selectedValue as boolean type

Reply:
Good suggestion, done!